projects/plugins/jetpack/tests/php/json-api/test-class.json-api-jetpack-site-endpoints.php

```diff
@@ -53,7 +63,44 @@ public function test_get_site() {
 
 		wp_set_current_user( $admin->ID );
 
-		$endpoint = new WPCOM_JSON_API_GET_Site_Endpoint(
+		$endpoint = $this->create_get_site_endpoint();
+
+		$response = $endpoint->callback( '', $blog_id );
+
+		$this->assertTrue( $response['jetpack'] );
+		$this->assertTrue( $response['jetpack_connection'] );
+
+		$options = (array) $response['options'];
+		$this->assertArrayHasKey( 'jetpack_connection_active_plugins', $options );
+	}
+
+	/**
+	 * Unit test for the `/sites/%s` endpoint.
+	 */
+	public function test_get_site_trials_list() {
+		global $blog_id;
+
+		// Current trials.
+		$trials = WPCOM_JSON_API_GET_Site_Endpoint::$jetpack_enabled_trials;
+
+		$admin = self::factory()->user->create_and_get(
+			array(
+				'role' => 'administrator',
+			)
+		);
+
+		wp_set_current_user( $admin->ID );
+
+		$endpoint = $this->create_get_site_endpoint();
```

Comment:
Is it possible to add a truthy test here as well? I'm still learning the ropes here but I believe it's possible to create a site with a migration trial now.

Reply:
It checks `false` because usually in Jetpack unit tests, all the functions that are wpcom-specific return `false` and nothing more. In this case, `has_blog_sticker`.

But I can change the function and make it more smart.